packages/nextjs/src/index.client.ts

```diff
@@ -9,7 +9,7 @@ import { addIntegration, UserIntegrations } from './utils/userIntegrations';
 export * from '@sentry/react';
 export { nextRouterInstrumentation } from './performance/client';
 
-const { BrowserTracing } = TracingIntegrations;
+const { BrowserTracing } = TracingIntegrations || {};
 export const Integrations = { ...BrowserIntegrations, BrowserTracing };
```

Comment:
I'd like us to handle this better in the future, but for now, please just add this TODO and edit.

```suggestion
// TODO: The `|| {}` here is a workaround to enable people not using tracing to reduce bundle size by using
// `resolve.alias["@sentry/tracing"] = false` in their webpack config to tell webpack 5+ to replace the tracing package
// with an empty object. If they do that , `TracingIntegrations` will be undefined and this destructuring will fail
// without the empty object being provided as an alternative from which to unpack. In the long run, we should handle
// this internally.
const { BrowserTracing } = TracingIntegrations || {};
export const Integrations = { ...BrowserIntegrations, ...(BrowserTracing && {BrowserTracing}) };
```